vc/libxmp.vcproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <VisualStudioProject
 	ProjectType="Visual C++"
-	Version="8.00"
+	Version="8,00"
```

Comment:
Apologies -- this `.` to `,` change seems to be because of my windows regional setting and is not necessary.